### DISCUSSION TITLE
Getting PayPal Express Checkout working again?

### DISCUSSION THREAD

**Discussion by Galane** [zencart/zencart#6510] [Q&A]
I'm working on a site using old 1.3.9h and at some point it quit allowing Guest Checkout with PayPal despite the customer's PayPal account having the option enabled. How can this be fixed? The customer does not want to change/update anything with the site, except for having it not force people to get a PayPal account to pay.

Also, what's up with the Zen-Cart Forum instantly banning a new login without giving any reason? Does it just ban all Yahoo and Gmail email addresses? Who do I contact to get that fixed?

**Comment by scottcwilson** [comment]
Sometimes the anti-spam is a bit overzealous.  What's your forum userid, I will fix.

**Comment by Galane** [comment]
EMCparts

I tried doing a new account with my gmail address ***@***.*** and got the instant ban notice. Go ahead and delete that one. Keep the one with ***@***.***

I've had this Yahoo Mail account since back when they gave users two megabytes of mail storage, *double* the one megabyte Hotmail had.

On Friday, May 31, 2024 at 05:04:28 PM MDT, Scott C Wilson ***@***.***> wrote: 

Sometimes the anti-spam is a bit overzealous. What's your forum userid, I will fix.

**Comment by scottcwilson** [comment]
OK you should be able to login now.  Email to the address associated with this account was bouncing back, that's why your account was closed.  Please check this and get it fixed.

**Comment by Galane** [comment]
Nothing I can do about the occasional Yahoo outage or glitch, same as with any mail service. The forum had no issue sending me the password reset emails.


On Saturday, June 1, 2024 at 04:25:21 AM MDT, Scott C Wilson ***@***.***> wrote: 

OK you should be able to login now. Email to the address associated with this account was bouncing back, that's why your account was closed. Please check this and get it fixed.
